### ISSUE TITLE
HTML editor not word processor

### ISSUE THREAD

**Issue by blue-dreamer** [ckeditor/ckeditor5#9192]
## 📝 Provide a description of the new feature

Most WYSIWYG's have always looked like word processors, not HTML formatting editors. I've always believed this to be misleading, this leads clients to think HTML documents are like Word documents and that can miss out important accessible markup.

Formatting toolbars are woefully inadequate for producing good accessible markup for a number of reasons:

1. Bold or Strong?
There's a subtle difference between the two, the `B` tag is visual only, `STRONG` is bold emphasis.
We should have formatting buttons available for both.

2. Italics or Emphasis
Like bold/strong, the `I` tag is visual only, the `EM` tag is emphasis
We should have formatting buttons available for both.

3. What about other commonly used HTML elements such as `INS` and `DEL`, `SMALL`, `CITE` ?

In short if CK is a tool for editing HTML markup then the correct tools should be provided as default. Yes you can add custom options but that rather defeats the object of getting clients to add accessibility to their content.

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by Reinmar** [comment]
Hi! Thanks for the feedback.

There's definitely miscommunication regarding what CKEditor 4 or 5 are. We do use the term "HTML WYSIWYG editor", however, we were forced to do that mainly by the fact that people use this term. I know that this phrase is vague and everyone expects a different thing from it.

Anyway, **CKEditor is definitely not an editor for any arbitrary HTML**. It's a structured content editor and HTML is one of its possible outputs. It's not an editor for developers that may (and want) to care about things like what is `<b>` and what is `<strong>`. In fact, even most developers don't care.

We did an extensive research on what markup best represents certain styling/semantic of textual content: [https://ckeditor.github.io/editor-recommendations/.](https://ckeditor.github.io/editor-recommendations/.) The currently used markup is a result of that. But our focus is on UX for non-developers.

> In short if CK is a tool for editing HTML markup then the correct tools should be provided as default.

So, tl;dr – it's not :) 

There's miscommunication caused mainly by ambiguity of commonly used terms.
